### PR TITLE
Fix for CI running the 2_ports_CREATE_test_traffic tests

### PR DESCRIPTION
**Pull request by zzxgzgz**
This PR attemps to do the following:

1. Enable the CI to run the 2_ports_CREATE_test_traffic tests, with docker network created, fixed IPs assigned to the two containers, and the 2_ports_CREATE_test_traffic tests called with the above IPs.

### CODE REVIEW

File: test/travis_test/build_2_container.sh [hunk 1]
```diff
@@ -42,8 +42,14 @@ if [ "$1" == "compile_and_run_unit_test" ]; then
 
 elif [ "$1" == "2_port_test_traffic" ]; then
   echo "    --- 2_ports_CREATE test case---"
+  parent_container_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' aca_PARENT)
+  child_container_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' aca_CHILD)
+  echo "aca_PARENT ip: $parent_container_ip"
+  echo "aca_CHILD ip: $child_container_ip"
+  docker exec aca_CHILD -c "/etc/init.d/openvswitch-switch restart && ovs-vswitchd --pidfile --detach" || true
```
Comment (er1cthe0ne) [inline test/travis_test/build_2_container.sh:49]: this is done in line 31 already, do we need to do it again here?

File: test/travis_test/build_2_container.sh [hunk 2]
```diff
@@ -42,8 +42,14 @@ if [ "$1" == "compile_and_run_unit_test" ]; then
...
+  docker exec aca_PARENT -c "/etc/init.d/openvswitch-switch restart && ovs-vswitchd --pidfile --detach" || true
```
Comment (er1cthe0ne) [inline test/travis_test/build_2_container.sh:50]: this is done in line 26 already, do we need to do it again here?

Comment (zzxgzgz) [reply test/travis_test/build_2_container.sh]: Removed these redundant lines and tested locally, the tests passed. Pushed the modified code. Please wait for the CI result. Thanks.

**Review by er1cthe0ne — APPROVED** [review]
Looks good. We have basic traffic test in CI now with this PR. I am happy :)

**Pull request merged**